code/game/objects/items/binoculars.dm

```diff
@@ -52,6 +56,14 @@
 			to_chat(usr, "<span class='notice'>They are currently set to railgun targeting mode.</span>")
 		if(MODE_ORBITAL)
 			to_chat(usr, "<span class='notice'>They are currently set to orbital bombardment mode.</span>")
+		if(MODE_MORTAR)
+			to_chat(usr, "<span class='notice'>They are currently set to mortar targeting mode.</span>")
+	if(!modes.Find(MODE_MORTAR))
+		return
```

Comment:
What kind of binoc won't have this mode?

Reply:
Default Range finder, then again i made it that way cuz what if someone wanted to make binocs without mortar capabilities